com.unity.render-pipelines.high-definition/Tests/Editor/BuildProcessors/HDRPPreprocessShaderTests.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using UnityEditor;
+using UnityEditor.Build;
+using NUnit.Framework;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace UnityEditor.Rendering.HighDefinition.Tests
+{
+    public class HDRPPreprocessShadersTests
```

Comment:
There is a stripping case (StripHDRPShader_WhenHDRPisNotCurrentPipeline).
For completeness, a non stripping case would be nice: DontStripHDRPShader_WhenHDRPisCurrentPipeline